pkg/rotator/rotator.go

```diff
@@ -778,7 +788,7 @@ func (r *ReconcileWH) Reconcile(ctx context.Context, request reconcile.Request)
 		artifacts, err := buildArtifactsFromSecret(secret)
 		if err != nil {
 			crLog.Error(err, "secret is not well-formed, cannot update webhook configurations")
-			return reconcile.Result{}, nil
+			return reconcile.Result{}, err
```

Comment:
I've found this error too. If the reconciliation has failed, we should reschedule it manually or just return the error for auto rescheduling it